docs/sources/tempo/community/_index.md

```diff
@@ -4,16 +4,23 @@ description: Participate in and contribute to the Tempo open source community.
 weight: 900
 ---
 
+# Community
+
+If you are interested in distributed tracing, consider participating and contributing to the [open source Tempo project](https://github.com/grafana/tempo).
+
+Want to learn more about open source communities? Read David Allen's blog post, [What makes a good open source community?](/blog/2023/09/01/what-makes-a-good-open-source-community/) on the Grafana website.
+
 ## Communicate
 
 - [Grafana Slack](https://slack.grafana.com/) #tempo channel
 - [Community Forum](https://community.grafana.com/c/grafana-tempo/40) - for questions/feedback.
 - [Community Call](https://docs.google.com/document/d/1yGsI6ywU-PxZBjmq3p3vAXr9g5yBXSDk4NU8LGo8qeY/edit#) - Monthly
-  on the second thursday at 1630 UTC.
+  on the second Thursday at 1630 UTC.
   Recordings available [online](https://www.youtube.com/playlist?list=PLDGkOdUX1Ujqe8WZ8T1h2pNjpll0t-KLw).
 - [Google Groups](https://groups.google.com/forum/#!forum/tempo-users)
```

Comment:
Should this line be removed? The last announcement to a google group was Tempo v0.5.
```suggestion
```